task-json/q2.js

```diff
@@ -0,0 +1,13 @@
+const timeinseconds = [4,5,2,10,0];
+
+timeinseconds.forEach(function delay (seconds){
+        return new Promise(function(resolve, reject) {
+  
+            setTimeout(resolve, seconds*100);
+        }).then(function() {
+            console.log(`${seconds} seconds` );
+        });
+    
+   
+   

```

Comment:
this is not a clean way to write a code
1. create delay function before loop
2. the loop is not as requested, please double check it